presto-product-tests/conf/tempto/tempto-configuration-for-docker-tls-kerberos.yaml

```diff
@@ -0,0 +1,48 @@
+hdfs:
```

Comment:
do you need to define new tempto configuration file, I thought you reuse file which is used other kerberized environment